scripts/setupPythonEnvironment.bash

```diff
@@ -69,6 +70,11 @@ shift # past argument or value
 done
 
 
+# Clone the package directory
+echo "Cloning the GEOS python package repository..."
+git clone --depth 1 --branch main https://github.com/GEOS-DEV/geosPythonPackages.git $PACKAGE_DIR
```

Comment:
I possible, I would put the clone very next to the definition of `PACKAGE_DIR`? That way, we can connect to the `TARGET_PACKAGES` as well?
Also, so you want to add `--single-branch`?

Reply:
Sounds good.  I've restructured the setup script to match this behavior